samples/QuickstartClient/Program.cs

```diff
@@ -12,7 +12,7 @@
 
 var (command, arguments) = GetCommandAndArguments(args);
 
-var clientTransport = new StdioClientTransport(new()
+var clientTransport = new StdioClientTransport(new
```

Comment:
You can't remove parens in this case. When the type name isn't named, it becomes the creation of an anonymous type.

Reply:
Which for whatever reason is still succeeding on my machine.

Reply:
Which is because `dotnet test` apparently only builds the transitive closure of test projects. TIL.